packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -52,7 +52,7 @@ def test_if_all_exhibitors_are_in_sync(dcos_api_session):
     correct_data = sorted(r.json(), key=lambda k: k['hostname'])
 
     for zk_ip in dcos_api_session.masters:
-        resp = requests.get('http://{}:8181/exhibitor/v1/cluster/status'.format(zk_ip))
+        resp = requests.get('http://{}/exhibitor/exhibitor/v1/cluster/status'.format(zk_ip), verify=False)
```

Comment:
Please add a comment that this test method relies on the fact that the Admin Router instance being talked to here proxies the HTTP request to the _local_ Exhibitor instance, on the same node.

For posterity: the goal here is to test a specific Exhibitor instance. The patch makes sure that we reach that through Admin Router, instead of directly reaching it. But we must still ensure that we test _all_ Exhibitor instances. That is currently guaranteed precisely because Admin Router proxies to the local Exhibitor (as opposed to proxying to e.g. `https://master.mesos:8181` which would contact a "random" Exhibitor.

Reply:
Let's please also change the variable name `zk_ip` to `master_node_ip`.

Reply:
Done.